runtime/src/iree/vm/bytecode/dispatch_util.h

```diff
@@ -296,7 +296,7 @@ static inline const iree_vm_register_list_t* VM_DecVariadicOperandsImpl(
   }
 
 #define DISPATCH_OP(ext, op_name, body)                               \
-  _dispatch_##ext##_##op_name:;                                       \
+  _dispatch_##ext##_##op_name :;                                      \
```

Comment:
@benvanik should this be changed?

Reply:
that semicolon may be safe to remove - I think I know why I added it (working around _other_ clang-format issues), but don't think I needed to and I can clean it up later - all the other improvements in these files are better :)